x-pack/legacy/plugins/ml/common/constants/index_patterns.ts

```diff
@@ -10,3 +10,4 @@ export const ML_ANNOTATIONS_INDEX_PATTERN = '.ml-annotations-6';
 
 export const ML_RESULTS_INDEX_PATTERN = '.ml-anomalies-*';
 export const ML_NOTIFICATION_INDEX_PATTERN = '.ml-notifications';
+export const ML_DF_NOTIFICATION_INDEX_PATTERN = '.data-frame-notifications-1';
```

Comment:
Just double checking, but do the data frame notifications always go in `.data-frame-notifications-1` (with the `1`)?

Reply:
Yes - that's the index it will go into